backend/src/db/migrations/23-video-block-show-link.sql

```diff
@@ -0,0 +1,10 @@
+-- Adds a single `show_link` field to blocks, used for video blocks to also show
+-- a link to the corresponding video page. We also drop and re-add the appropriate constraint.
+alter table blocks
+    add column show_link boolean default false,
+    drop constraint video_block_has_fields,
+    add constraint video_block_has_fields check (type <> 'video' or (
+        show_title is not null and
+        show_link is not null
+    ));
+    
```

Comment:
Still wrong unfortunately :D The last line now contains whitespace, so the file ends with whitespace. But it has to end with a newline. I would recommend configuring your editor somehow to do this automatically.

Reply:
I don't know why it did that, it's not like I hit the spacebar there a couple of times or sth 😕